NoComponentDrag.py

```diff
@@ -168,13 +165,12 @@ def run(context):
                                                                  f'Component Drag',
                                                                  'Enables or disables the movement of components by dragging '
                                                                   'in the canvas.\n\n'
-                                                                  f'({NAME} v {manifest_["version"]})',
+                                                                  f'({NAME} v {manifest_["version"]})\n',
```

Comment:
All of fusions text boxes tooltip thingys have a spare line at the end to seperate the text from the bottom of the box. This looks okay but I might actually prefer omitting the line break and keeping it as it was.